.github/workflows/sync-version-with-upstream.yml

```diff
@@ -14,14 +14,14 @@ concurrency:
 jobs:
   sync:
     name: 🔄 Sync version with upstream
-    environment: "Candidate Branch"
+    environment: "LTS Branch"
     runs-on: ubuntu-latest
     steps:
       - name: 🔄 Sync version with upstream
         uses: snapcrafters/ci/sync-version@main
         with:
           token: ${{ secrets.SNAPCRAFTERS_BOT_COMMIT }}
           update-script: |
-            # Grabbing latest release
-            VERSION=$(curl -sL https://api.github.com/repos/jenkinsci/jenkins/releases |  jq .[].tag_name -r | grep -v rc | sort -r | head -n 1 | cut -d "-" -f2)
+            # Grabbing latest lts release
+            VERSION=$(curl -sL https://api.github.com/repos/jenkinsci/jenkins/releases |  jq .[].tag_name -r | grep -v rc | sort -r | grep "2.440" | head -n1 | cut -d "-" -f2)
             sed -i 's/^\(version: \).*$/\1'"'$VERSION'"'/' snap/snapcraft.yaml 
```

Comment:
This file can be removed, and you'll need to add another stanza to the update script on the `candidate` branch, because scheduled workflows only run on the default branch. See sublime-text or sublime-merge for an example :)